src/components/table/OcTableFiles.vue

```diff
@@ -23,7 +23,7 @@
     <template #select="{ item }">
       <oc-checkbox
         :id="`oc-table-files-select-${item.id}`"
-        :label="$gettext('Select')"
+        :label="$gettext('Select resource')"
```

Comment:
The item has information available whether it's a folder or not. Could you go the extra mile an introduce a computed prop which has `$gettext('Select folder')` for a folder and `$gettext('Select file')` otherwise? Context: we had some usability tests recently and they unveiled that it's not directly clear what we mean by `resource`.

Reply:
Property of the item is `type` and has the value `folder` in case of a folder.

Reply:
Great idea :) I solved this via a method though, because we need to pass `resource` as parameter.

Reply:
True 🚀 nice!